build.gradle

```diff
@@ -50,6 +50,7 @@ dependencies {
 	compile 'commons-codec:commons-codec:1.10'
 	compile 'commons-httpclient:commons-httpclient:3.1'
 	compile 'commons-logging:commons-logging:1.2'
+	compile 'commons-io:commons-io:2.4'
```

Comment:
If we are no longer going to add the jars to the project as well then we should remove all unnecessary jars from the lib directory. Otherwise we are in some weird in between state.